### PR TITLE
Image refresh for rhel-8-4

### PR DESCRIPTION
**Pull request by martinpitt**
This is a copy of PR #1462, I can't push to that cockpituous branch for some reason.

### CODE REVIEW

**Review by marusak — APPROVED** [review]
Thanks!

**Comment by martinpitt** [comment]
Bwah! The naughty [doesn't match somehow](https://logs.cockpit-project.org/logs/pull-1468-20201215-152631-fd5dfe6f-rhel-8-4-cockpit-project-cockpit/log.html#281-2), so seems I need to spend more than half a brain cell on this :-/

File: naughty/rhel-8/1453-fsadm-unbound-variable [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/sbin/fsadm: line *: $3: unbound variable
```
Comment (marusak) [inline naughty/rhel-8/1453-fsadm-unbound-variable:1]: `warning: /sbin/fsadm: line 815: $3: unbound variable`
just the path is wrong, not `/usr/bin/` but `/sbin/`

Comment (martinpitt) [reply naughty/rhel-8/1453-fsadm-unbound-variable:1]: Hah, thanks for spotting! Pushed.

Comment (martinpitt) [reply naughty/rhel-8/1453-fsadm-unbound-variable]: Yay, worked now!

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**